tests/cypress/e2e/Teacher page/Manual/clickingytlink.cy.js

```diff
@@ -0,0 +1,19 @@
+import {loginForAdmin, loginForTeacher} from '../../tools/login/login.js'
+import { goToHome, goToLogin, goToRegister, goToPage } from "../../tools/navigation/nav.js";
+
+describe('Is able to see teacher page', () => {
+  it('Passes', () => {
+    loginForTeacher();
+    cy.wait(500);
+    goToPage('/for-teachers/manual');
+    cy.get('#section-1 > :nth-child(3) > a').should('have.attr', 'href').and('include', 'https://www.youtube.com/watch?v=EdqT313rM40&t=2s');
+
+
+//    cy.location().then((loc) => {
```

Comment:
Same here, and I think in all other files, could you remove these or is there a different action to be taken (sometimes commented out code is a "note to self")

Reply:
Good point. All commented out code has just been removed.